paymentsheet/src/main/java/com/stripe/android/paymentsheet/ui/PromoBadge.kt

```diff
@@ -86,6 +103,18 @@ private fun FixedTextSize(
     }
 }
 
+private val Context.isEnglishLanguage: Boolean
+    get() {
+        val currentLocale: Locale = if (SDK_INT >= N) {
+            resources.configuration.locales[0]
+        } else {
+            @Suppress("DEPRECATION")
+            resources.configuration.locale
+        }
+
+        return currentLocale.language == Locale.ENGLISH.language
+    }
```

Comment:
[nit] Likely useful on other scenarios, could be moved to a common module

Reply:
I honestly don’t want to further encourage this pattern, so I’ll keep it here for now 😓